.github/ISSUE_TEMPLATE/01_explorer_bug.yaml

```diff
@@ -0,0 +1,34 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Explorer bug
+description: >
+  Report a bug with the explorer interpreter. This is what's provided at
+  carbon.compiler-explorer.com.
+labels: explorer
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **Attention:** If this is a _question_, please use either [GitHub
+        Discussions](https://github.com/carbon-language/carbon-lang/discussions)
+        or [Discord](https://discord.gg/ZjVdShJDAs).
+  - type: textarea
+    id: desc
+    attributes:
+      label: >
+        Description of the bug:
+  - type: textarea
+    id: repro
+    attributes:
+      label: >
+        What's the simplest, easiest way to reproduce this bug?
+      description: >
+        Please provide example code and errors; a shortlink to the execution on
+        carbon.compiler-explorer.com can also help.
```

Comment:
I wonder if the structure of "What did you do?", "What did you expect to happen?" "What actually happened?" would be more likely to give us good information here. I could imagine someone filling out this form without ever saying why they think the observed behavior is a bug.

Reply:
Done